content/installation/Webhook/_index.md

```diff
@@ -22,11 +22,11 @@ BotKube can be integrated with external apps via Webhooks. A webhook is essentia
 - Deploy BotKube backend using **helm install** in your cluster:
 
   ```bash
```

Comment:
I would like to suggest additional changes in this section:
``````markdown
#### Using Helm

- We will be using [helm](https://helm.sh/) to install BotKube in Kubernetes. Follow [this](https://docs.helm.sh/using_helm/#installing-helm) guide to install helm if you don't have it installed already.
- Add **botkube** chart repository:

  ```bash
  helm repo add botkube https://charts.botkube.io
  helm repo update
  ```

- Deploy BotKube backend using **helm install** in your cluster:

  ```bash
  export CLUSTER_NAME={cluster_name}
  export WEBHOOK_URL={url}

  helm install --version v0.13 botkube --namespace botkube --create-namespace \
  --set communications.default-group.webhook.enabled=true \
  --set communications.default-group.webhook.url=${WEBHOOK_URL} \
  --set settings.clusterName=${CLUSTER_NAME} \
  botkube/botkube
  ```
``````

Reasons:
1. Have `export` at the beginning of the `helm install`. It's easier to set them and see also what needs to be set instead of looking into multiple `--set` statements.
2. The `image.tag` is not needed as it's by default like that in the release Helm chart
3. The `--set config.settings.clustername` must be changed to `--set settings.clusterName`

Thanks!

Reply:
Sure, addressed